src/renderer/features/lyrics/lyrics.tsx

```diff
@@ -0,0 +1,131 @@
+import { useEffect, useRef, useState } from 'react';
+import isElectron from 'is-electron';
+import { ErrorBoundary } from 'react-error-boundary';
+import { ErrorFallback } from '/@/renderer/features/action-required';
+import { useCurrentServer, useCurrentSong } from '/@/renderer/store';
+import { SynchronizedLyrics } from './synchronized-lyrics';
+import { UnsynchronizedLyrics } from '/@/renderer/features/lyrics/unsynchronized-lyrics';
+import { LyricLine } from '/@/renderer/features/lyrics/lyric-line';
+import { Center, Group } from '@mantine/core';
+import { RiInformationFill } from 'react-icons/ri';
+import { TextTitle } from '/@/renderer/components';
+import { SynchronizedLyricsArray } from '/@/renderer/api/types';
+import { useSongLyrics } from '/@/renderer/features/lyrics/queries/lyric-query';
+
+const lyrics = isElectron() ? window.electron.lyrics : null;
+
+const ipc = isElectron() ? window.electron.ipc : null;
+
+// use by https://github.com/ustbhuangyi/lyric-parser
+const timeExp = /\[(\d{2,}):(\d{2})(?:\.(\d{2,3}))?]([^\n]+)\n/g;
+
+export const Lyrics = () => {
+  const currentServer = useCurrentServer();
```

Comment:
The queue is independent of the current server, meaning you'll need to use `serverId` from the `currentSong` to determine what the actual server is.